src/entity/object/Painting.php

```diff
@@ -40,7 +41,7 @@
 use pocketmine\world\World;
 use function ceil;
 
-class Painting extends Entity{
+class Painting extends Entity implements FacingInterface{
```

Comment:
Design error, you're using an interface made for blocks in an entity, I don't think you should do that